wiki/Play_Styles/Drag/en.md

```diff
@@ -0,0 +1,20 @@
+---
+tags:
+  - tablet
+  - dragging
+  - play style
+---
+
+<!-- This article is a stub -->
+
+# Drag
+
+*Main Page: [Play Styles](/wiki/Play_Styles)*
+
+**Drag** is a term used to describe the act of navigating (or "aiming") a [tablet](/wiki/Glossary#tablet) pen by physically touching the pen to the surface of the tablet and "dragging" it to the desired location (as opposed to [hovering](/wiki/Play_Styles/Hover)). This style is used by players of all skill levels and is easy to learn.
+
+Arguments in-favor of dragging often claim that it is more comfortable/natural and is much harder to "overshoot" on [jumps](/wiki/Beatmaps/Pattern/Jump). 
+
+By the same token, many players that argue against dragging claim that it can damage the surface of the tablet without the use of a tablet cover, requires purchasing replacement nibs (due to nib wear), and that the friction is (techincally) "slower".
```

Comment:
```suggestion
By the same token, many players that argue against dragging claim that it can damage the surface of the tablet without the use of a tablet cover, requires purchasing replacement nibs (due to nib wear), and that the friction is (technically) "slower".
```
fix typo